dev/com.ibm.ws.security.authentication.builtin/src/com/ibm/ws/security/authentication/jaas/modules/LoginModuleHelper.java

```diff
@@ -1,28 +1,37 @@
 /*******************************************************************************
- * Copyright (c) 2022 IBM Corporation and others.
+ * Copyright (c) 2022, 2024 IBM Corporation and others.
  * All rights reserved. This program and the accompanying materials
  * are made available under the terms of the Eclipse Public License 2.0
  * which accompanies this distribution, and is available at
  * http://www.eclipse.org/legal/epl-2.0/
- * 
+ *
  * SPDX-License-Identifier: EPL-2.0
  *
  * Contributors:
  *     IBM Corporation - initial API and implementation
  *******************************************************************************/
 package com.ibm.ws.security.authentication.jaas.modules;
 
-import org.osgi.framework.BundleContext;
-import org.osgi.framework.FrameworkUtil;
-
+import com.ibm.ws.kernel.service.util.ServiceCaller;
 import com.ibm.ws.security.authentication.internal.JAASService;
-import com.ibm.wsspi.security.common.auth.module.IdentityAssertionLoginModule;
 
 /**
  *
  */
 public class LoginModuleHelper {
     private static JAASService testJaasService = null;
+    // Unit test require us not to fail to construct ServiceCaller
+    private static final ServiceCaller<JAASService> jaasService;
+    static {
+        ServiceCaller<JAASService> temp;
+        try {
+            temp = new ServiceCaller<>(LoginModuleHelper.class, JAASService.class);
+        } catch (IllegalStateException e) {
```

Comment:
Looks good.  ServiceCaller(<init>) should never throw NPE as neither parameter can be null.